src/test/java/org/broadinstitute/hellbender/tools/sv/cluster/SVClusterEngineTest.java

```diff
@@ -400,15 +408,191 @@ public void testClusterTogetherCNVs() {
+                                                     final List<SVCallRecord.ComplexEventInterval> cpxIntervals, final boolean expected) {
+        final SVCallRecord cpx1 = new SVCallRecord("cpx1", "chr1", 1000, null,
+                "chr1", 2000, null,
+                GATKSVVCFConstants.StructuralVariantAnnotationType.CPX,
+                GATKSVVCFConstants.ComplexVariantSubtype.delINV,
+                Arrays.asList(new SVCallRecord.ComplexEventInterval("DEL_chr1:1100-1500"), new SVCallRecord.ComplexEventInterval("INV_chr1:1600-1900")),
+                null, Collections.singletonList(SVTestUtils.PESR_ALGORITHM),
+                Lists.newArrayList(Allele.REF_N, SVTestUtils.CPX_ALLELE),
+                Collections.emptyList(), Collections.emptyMap(), Collections.emptySet(), null, SVTestUtils.hg38Dict);
+        final SVCallRecord cpx2 = new SVCallRecord("cpx2", contigA, posA, null,
+                contigB, posB, null,
+                GATKSVVCFConstants.StructuralVariantAnnotationType.CPX,
+                subtype,
+                cpxIntervals,
+                null, Collections.singletonList(SVTestUtils.PESR_ALGORITHM),
+                Lists.newArrayList(Allele.REF_N, SVTestUtils.CPX_ALLELE),
+                Collections.emptyList(), Collections.emptyMap(), Collections.emptySet(), null, SVTestUtils.hg38Dict);
+        Assert.assertEquals(engine.getLinkage().areClusterable(cpx1, cpx2), expected);
+    }
+
+    @DataProvider(name = "testClusterTogetherInsertedComplexData")
+    public Object[][] testClusterTogetherInsertedComplexData() {
+        return new Object[][]{
+                // exact match
+                {"chr1", 1000, "chr1", 1000,
+                        GATKSVVCFConstants.ComplexVariantSubtype.dDUP,
+                        Arrays.asList(
+                                new SVCallRecord.ComplexEventInterval(GATKSVVCFConstants.StructuralVariantAnnotationType.DUP, new SimpleInterval("chr1", 6000, 8000))
+                        ),
+                        true
+                },
+                // close match
+                {"chr1", 1010, "chr1", 1010,
+                        GATKSVVCFConstants.ComplexVariantSubtype.dDUP,
+                        Arrays.asList(
+                                new SVCallRecord.ComplexEventInterval(GATKSVVCFConstants.StructuralVariantAnnotationType.DUP, new SimpleInterval("chr1", 6200, 7990))
+                        ),
+                        true
+                },
+                // not match by coordinates
+                {"chr1", 2000, "chr1", 3000,
+                        GATKSVVCFConstants.ComplexVariantSubtype.dDUP,
+                        Arrays.asList(
+                                new SVCallRecord.ComplexEventInterval(GATKSVVCFConstants.StructuralVariantAnnotationType.DUP, new SimpleInterval("chr1", 6200, 7990))
+                        ),
+                        false
+                },
+                // not match by subtype
```

Comment:
It would be good to update this to look like a real dDUP_iDEL for a more accurate test (END coordinate, adding DEL interval to CPX_INTERVALS)

Reply:
I agree that it would be better to not have bogus records, but I think for testing purposes this should be left the way it is. If the records differ in any other way than the subtype, we don't know for sure if `areClusterable()` is returning  false due to the intended reason.